sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2463,6 +2463,242 @@ async Task<Page<DocumentHealthcareResult>> NextPageFunc(string nextLink, int? pa
+        {
+            Argument.AssertNotNullOrEmpty(documents, nameof(documents));
+            Argument.AssertNotNull(options, nameof(options));
+            MultiLanguageBatchInput documentInputs = ConvertToMultiLanguageInputs(documents);
+
+            return StartAnalyzeOperationBatch(documentInputs, options, cancellationToken);
+        }
+
+        /// <summary>
+        /// Analyze Operation enables the application to have multiple tasks including NER, PII and KPE.
+        /// Accepts a list of strings which are analyzed asynchronously.
+        /// For document length limits, maximum batch size, and supported text encoding, see
+        /// <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits"/>.
+        /// </summary>
+        /// <param name="documents">The list of documents to analyze.</param>
+        /// <param name="options"> The different operations to pass as options.
+        /// You can use it to have multiple tasks to analyze as well as multiple task item per each individual task.
+        /// For example -
+        ///    AnalyzeOperationOptions operationOptions = new AnalyzeOperationOptions()
+        ///    {
+        ///        KeyPhrasesTaskParameters = new KeyPhrasesTaskParameters(),
+        ///        EntitiesTaskParameters = new EntitiesTaskParameters(),
+        ///        PiiTaskParameters = new PiiTaskParameters(),
+        ///        DisplayName = "AnalyzeOperation"
+        ///    };
+        /// By default ModelVersion is set as 'latest' and it can set from the task parameters.
+        /// KeyPhrasesTaskParameters = new KeyPhrasesTaskParameters()
+        /// {
+        ///     ModelVersion = "latest"
+        /// },
+        /// For additional configurable options see <see cref="AnalyzeOperationOptions"/> </param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <exception cref="RequestFailedException">Service returned a non-success
+        /// status code.</exception>
+        public virtual async Task<AnalyzeOperation> StartAnalyzeOperationBatchAsync(IEnumerable<TextDocumentInput> documents, AnalyzeOperationOptions options, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(documents, nameof(documents));
+            Argument.AssertNotNull(options, nameof(options));
+            MultiLanguageBatchInput documentInputs = ConvertToMultiLanguageInputs(documents);
+
+            return await StartAnalyzeOperationBatchAsync(documentInputs, options, cancellationToken).ConfigureAwait(false);
+        }
+
+        private AnalyzeOperation StartAnalyzeOperationBatch(MultiLanguageBatchInput batchInput, AnalyzeOperationOptions options, CancellationToken cancellationToken = default)
+        {
+            options ??= new AnalyzeOperationOptions();
```

Comment:
nit: because it is mandatory in all methods that call this one, this is not needed.

Reply:
same for the one in the async method